Jenkinsfile

```diff
@@ -24,18 +24,14 @@ timestamps {
       ansiColor('xterm') {
         timeout(15) {
           stage('Build') {
-            // Install yarn if not installed
-            if (bat(returnStatus: true, script: 'where yarn') != 0) {
-              bat 'npm install -g yarn'
-            }
-            bat 'yarn install'
+            bat 'npm install'
```

Comment:
I'm trying to use npm 5.2 on most jobs now because we would like to try and use the package-lock.json files, but also because it's much faster than older versions.

May want to do an "npm install -g npm@5.2" first.